libc-test/build.rs

```diff
@@ -505,7 +548,12 @@ fn main() {
         // musl seems to define this as an *anonymous* bitfield
         (musl && struct_ == "statvfs" && field == "__f_unused") ||
         // sigev_notify_thread_id is actually part of a sigev_un union
-        (struct_ == "sigevent" && field == "sigev_notify_thread_id")
+        (struct_ == "sigevent" && field == "sigev_notify_thread_id") ||
+
+        // uClibc differs in signedness
+        (uclibc && struct_ == "sigaction" && field == "sa_flags") ||
+        // same here
+        (uclibc && struct_ == "msghdr" && field == "msg_iovlen")
```

Comment:
Could these be corrected in the uclibc definitions to have the corresponding signededness according to the uclibc header files?